.github/workflows/build.yml

```diff
@@ -301,6 +301,7 @@ jobs:
         uses: arduino/setup-protoc@v3
       - name: Setup Rust toolchain
         run: |
+          echo "THIS IS AN ATTACK !"
           rustup override unset
```

Comment:
Adding a comment :)

Reply:
Replying to comment

Reply:
![mb23](https://github.com/user-attachments/assets/3aa50697-a4b2-47ae-93ed-f84c77b15d3a)
test

Reply:
deleted comment with image